app/templates/complete.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="exit-text text-center">
+  <p>{{t 'previouslogin.line1'}}</p>
```

Comment:
Feature question:

This text tells someone to contact their local ISP coordinator. Does the site provide any mechanism for contacting help? How does the user know who to contact, or how?